### ISSUE TITLE
Is there any way to display fps on ps5?

### ISSUE THREAD

**Issue by xlj44400** [illusion0001/libhijacker#11]
ps5 activation similar to ps4 hen fps counter

**Comment by Wolfitaft2000** [comment]
yep. hard to know what fliprate unlock effects... would be nice if there is a display on ps5 (on/off)

**Comment by HiagoPereira** [comment]
I'm having issues with Crash Team Racing Nitro Fueled. Both CUSA14876 and CUSA13795. Game works, sometimes u can even get to a race, but it will freeze and crash PS5. I think its a kernel panic, but yeah, it happens all the time
